tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -102,3 +104,32 @@ def objective(trial):
     study.optimize(objective, n_trials=1)
     assert study.trials[0].state == optuna.trial.TrialState.COMPLETE
     assert study.trials[0].value == 1.0
+
+
+@pytest.mark.parametrize("max_epochs", [6, 7])
```

Comment:
Just a nit. Could you follow the same style for testing s.t. https://github.com/optuna/optuna/pull/1161/files#diff-19ce543d124092cc29a1e3eaeeb8617bR12 ?

e.g.

```
@pytest.mark.parametrize("interval, max_epochs", [(7, 7), (8, 7), (8, 8)])
```

It is easier to understand, because we can find the definition `interval` easily.

Reply:
Done.